src/quo/components/buttons/button/view.cljs

```diff
@@ -37,14 +37,16 @@
                pressed? on-press-in on-press-out]
         :or   {type                :primary
                size                40
-               customization-color (cond (= type :primary)  :blue
-                                         (= type :positive) :success
-                                         (= type :danger)   :danger
-                                         :else              nil)}}
+               customization-color :blue}}
        children]
-      (let [{:keys [icon-color background-color label-color border-color blur-type
+      (let [customization-color-by-type (condp = type
+                                          :primary  customization-color
+                                          :positive :success
+                                          :danger   :danger
+                                          nil)
```

Comment:
IMO, we should fix this in the `properties.cljs` as customization-color is not applicable when it's not type `primary`. WDYT?

https://github.com/status-im/status-mobile/blob/61c0650ceca1d0d6f44fedfd34ccc34f8fe99e10/src/quo/components/buttons/button/properties.cljs#L81-L86

cc: @J-Son89